modules/ROOT/pages/index.adoc

```diff
@@ -11,8 +11,12 @@ The same editor also allows you to edit RAML and WSDL API specification files an
 
 == Requirements
 
-* Internet Explorer versions 11 and earlier are not supported. +
-Anypoint Studio uses your system's native browser to display elements of the Anypoint Platform UI. If your system defaults to an Internet Explorer browser, you won't be able to visualize any of the Anypoint Platform views.
+* Studio 6.x will miss some features in Windows due to the fact that Anypoint Platform no longer supports Internet Explorer (see xref:docs-general::browser-support.adoc[Browser Support for Anypoint Platform]).
+** Authentication over MFA.
+** Access Exchange web UI.
+** Deploy to CloudHub.
+
+As IE is no longer supported, other features may not work in the future.
```

Comment:
```suggestion
Because Internet Explorer is no longer supported, other features might not work in the future.
```

Reply:
(may implies permission; use might for probability)

Reply:
Like it! Thanks!